### DISCUSSION TITLE
Failed to initlalize the SSL layer.

### DISCUSSION THREAD

**Discussion by mikeydelamonde** [mobizt/ESP-Google-Sheet-Client#19] [Q&A] [answered]
Thank you for getting updating the library for the Arduino MKR Wifi 1010.

I have uncommented the rootCACert section, and now when it is running I get the following error. Any ideas? Almost working!

Do I need to generate my own Root Certificate? I can't see that in the instructions.

```
Append spreadsheet values...
----------------------------
> WARN.mRunUntil: Terminating because the ssl engine closed.
> ERROR.mConnectSSL: Failed to initlalize the SSL layer.
> ERROR.mConnectSSL: Validation time is unknown.
unknown error
```

**Comment by mobizt** [comment, marked as answer]
Thanks for reporting this issue.

Now it was fixed in [v1.4.3](https://github.com/mobizt/ESP-Google-Sheet-Client/releases/tag/v1.4.3) already, please update.

**Reply by mikeydelamonde** [reply]
Thank you so much! It's now working.. :)
